src/app/package/page.tsx

```diff
@@ -0,0 +1,129 @@
+'use client'
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableRow,
+} from '@mui/material'
+import { useGetPackageDetailQuery } from '@/redux/api/packageApi'
+import PackageCard from './PackageCard'
+import { PackageDetailResponse } from '@/types/package'
+import { capitalizeWords } from '@/utils/capitalizeWords'
+import { useGetLatestSubscriptionQuery } from '@/redux/api/subscriptionApi'
+import { LatestSubscriptionResponse } from '@/types/subscription'
+import { formatRupiah } from '@/utils/formatRupiah'
+import { formatDate } from '@/utils/formatDate'
+
+export default function PackageList() {
+  const { data: free_package = {} as PackageDetailResponse } =
+    useGetPackageDetailQuery(1)
+  const { data: premium_package = {} as PackageDetailResponse } =
+    useGetPackageDetailQuery(2)
```

Comment:
In my opinion, you can use variable for the number. I get what you are saying because 1 means package with id: 1. But what if the schema changes and id become UUID format, this will be hard to maintain, so you can refactor this to a constant.ts so that you don't have to change the main package list code.

constant.ts
```
export const PACKAGE = {
     "free": 1,
     "premium": 2
}
```

Then you can call these constant, such as `PACKAGE["free"]`

Feels free to have an opinion about my comments though!

Reply:
Have fixed! Please check 😃 @emiriko @JohannesSetiawan